src/main/java/lotto/model/LottoTickets.java

```diff
@@ -27,4 +28,9 @@ public int getSize() {
     public List<LottoTicket> getLottoTickets() {
         return Collections.unmodifiableList(lottoTickets);
     }
+
+    public LottoTickets add(LottoTickets additionalLottoTickets) {
+        lottoTickets.addAll(additionalLottoTickets.getLottoTickets());
+        return this;
```

Comment:
객체 자기 자신을 반환하는 대신에, 새로운 객체를 생성해서 반환해보면 어떨까요? 

```suggestion
    public LottoTickets add(LottoTickets additionalLottoTickets) {
        List<LottoTicket> newLottoTickets = new ArrayList<>(this.lottoTickets);
        newLottoTickets.addAll(additionalLottoTickets.lottoTickets);
        
        return new LottoTickets(newLottoTickets);
```